lib/tasks/osc.rake

```diff
@@ -74,6 +74,26 @@ namespace :osc do
     end
   end
 
+  def version_changed? updated_spec_file
+    begin
+      file = Tempfile.new('yast-rake')
+      file.close
+      sh "osc -A '#{obs_api}' cat '#{obs_sr_project}' '#{package_name}' '#{package_name}.spec' > #{file.path}"
```

Comment:
What if the `obs_sr_project` does not contain the package yet? Imagine you create a completely new package which does not exist in Factory yet. Can you still build the tarball for the initial submission?